### PR TITLE
feat: added belgium flag

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
# Zero To Mastery - Canvas 2022 🎉🎉

## PR contains:
- Flag of Belgium

## Breaking changes
- None

## Screenshots
icon

### CODE REVIEW

File: src/art/Mohasin Haque/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>France Flag</title>
```
Comment (sophiabrandt) [inline src/art/Mohasin Haque/index.html:7]: France Flag???

File: src/art/Mohasin Haque/index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <div class="container">
+      <canvas id="franceFlagCanvas" width="400" height="300"></canvas>
```
Comment (sophiabrandt) [inline src/art/Mohasin Haque/index.html:12]: Canvas id also contains France

File: src/art/Mohasin Haque/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+let canvas = document.getElementById("franceFlagCanvas");
```
Comment (sophiabrandt) [inline src/art/Mohasin Haque/index.js:1]: Same here

File: src/art/avatar/index.html [hunk 4]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Belgium Flag</title>
 </head>
```
Comment (sophiabrandt) [inline src/art/avatar/index.html:8]: Please explain why you change another committer's code.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please check your pull request for code quality.

**Comment by MattCSmith** [comment]
Closing due to inactivity on requested changed. Feel free to reopen and address

**Comment by Mohasin-Haque** [comment]
@MattCSmith Please reopen this.. I have fixed the issues and ready to commit here

File: src/art/avatar/index.html (hunk 4 above)
Comment (Mohasin-Haque) [reply src/art/avatar/index.html:8]: I fixed the issues. Kindly reopen this PR so that I can commit

**Pull request closed without merging**